Tools/SandboxTest.ps1

```diff
@@ -16,6 +16,18 @@ if ($validationResult -like '*Manifest validation failed.*') {
   throw 'Manifest validation failed.'
 }
 
+# Check if Windows Sandbox is enabled
+
+if (-Not (Test-Path "$env:windir\System32\WindowsSandbox.exe")) {
+  Write-Error -Category NotInstalled -Message @'
+Windows Sandbox does not seem to be avaiable. Check the following url for prerequisites and further details:    
+https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-sandbox/windows-sandbox-overview
+  
+You can run the following command in an elevated PowerShell for enabling Windows Sandbox:
+Get-WindowsOptionalFeature -Online -FeatureName 'Containers-DisposableClientVM'

```

Comment:
The command to enable it would be `Enable-WindowsOptionalFeature`. And there's a small typo: avaiable.

Reply:
That's right. Sorry.

Reply:
That's right. Sorry.